cmd/bolt-agent/main.go

```diff
@@ -330,6 +333,19 @@ func getApp() *cli.App {
 			Value:  0,
 			Hidden: true,
 		},
+		&cli.BoolFlag{
+			Name:  "actions",
+			Usage: "Allow execution of actions defined in the bolt app (beta feature)",
+		},
+		&cli.BoolFlag{
```

Comment:
I tried to name flags without `-` because else there is always the confusion where the dash comes, so I have names like `preferipv4` `userest`. Not saying this is the best approach just that this is inconsistent now.